docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md

```diff
@@ -187,6 +196,12 @@ process.
 To successfully authenticate wth Entra ID, you should use the `v2.0` API. This means that
 the `CAMUNDA_IDENTITY_ISSUER_BACKEND_URL` value should end with `/v2.0`.
 
+It's also important to follow the [steps described here](https://learn.microsoft.com/en-us/entra/identity-platform/reference-app-manifest#configure-the-app-manifest) to configure the app manifest and set the [accesstokenAcceptedVersion](https://learn.microsoft.com/en-us/entra/identity-platform/reference-app-manifest#accesstokenacceptedversion-attribute) to `2` like so:
+
+```json
+	"accessTokenAcceptedVersion": 2,
+```
+
```

Comment:
I wonder if in here, although its not directly related to this PR, we could add a note to say that the Identity UI is not available in this version? We can then remove this for 8.5 which is when the Identity UI is being introduced?

Reply:
@Ben-Sheppard can you add that in a separate PR or issue? It would be up to you and the Identity team to remember to remove it for 8.5. Is there an epic you can associate this with?

Reply:
Yea sure, I can raise a PR and then track it